compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc

```diff
@@ -0,0 +1,72 @@
+== Scan Images in GitLab Container Registry
+
+Configure Prisma Cloud to scan your GitLab Container Registry on GitLab EE without using administrator credentials.
+You can use GitLab Personal access token method to authenticate Prisma to access the GitLab Container Registry to manage and get a full list of all container registries/images.
+
+[.task]
+=== Create a New Registry Scan
+
+*Prerequisites*
+
+* xref:../../install/deploy-defender/defender_types.adoc[Install a Defender] somewhere in your environment.
+* Enable the GitLab Container Registry under your GitLab project settings if it's not enabled already. See the https://docs.gitlab.com/ee/administration/packages/container_registry.html[GitLab administrator documentation] to enable GitLab Container Registry across your GitLab instance.
+* Create a https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#personal-access-token-scopes[GitLab Personal access token] with at least "read_api" scope permission.
+* Create xref:../../authentication/credentials-store/gitlab-credentials.adoc[GitLab credentials using API token].
+
+[.procedure]
+. Log in to Console, and select *Defend > Vulnerabilities > Images > Registry settings*.
+
+. Select *Add registry*.
++
+image::gitlab-container-registry.png[scale=60]
+
+. In *Add New Registry*, enter the following values:
+
+.. In *Version*, select *GitLab Container Registry*.
+
+.. In *Registry*, enter your custom domain URL address to scan. If you don't have a custom GitLab domain, enter the URL as `https://registry.gitlab.com`.
+
+.. In *Repository*, enter the name of the repository to scan, or leave this blank to scan all repositories.
+
+.. Optionally enter the *Repositories to exclude* them from being scanned.
+
+.. In *Repository types*, select the repository types that Prisma Cloud should scan.
+
+.. Enter *Tag* numbers to scan, leave blank, or enter a wildcard (*) to scan all the tags.
+
+.. Optionally, enter *Tags to exclude*, to avoid scanning images with specified tags.
+
+.. Provide at least one of the fields:
+... *User ID*: GitLab user account. The user ID is used to get all registries associated with the user.
+... *Group ID*: Enter a single group ID, or a list of group IDs. The group ID is used to locate all the registries within a specific group.
+... *Project ID*: Enter a GitLab Project ID, or a list of project IDs. The project ID is used to locate all the registries located within a specific project.
++
+[NOTE]
+====
+- To trigger a full scan, including all repositories of a user across every group and project, enter just the user ID.
+- When you enter all the IDs, Prisma will query the project IDs and the group IDs, the user ID won't be queried.
+- When you enter both user ID and the group ID, Prisma will query the registries based on the group ID.
+- With both user ID and project ID, Prisma will query based on the project ID.
+- When both group ID and project ID are provided, Prisma queries both the IDs.
+====
```

Comment:
@Pubs-MV, The main use-case is to scan the entire repository. Should I include the other combinations or not! WDYT